docker/http-to-bucket/app.rb

```diff
@@ -131,3 +133,16 @@ def upload_response(project_id, bucket_name, object_name, http_response)
   # which will be JSON-encoded into a response.
   return JMESPath.search(jmespath, JSON.parse(http_response.body))
 end
+
+def clean(obj)
```

Comment:
This is a neat implementation. A couple of nitpicks:

* Could the name be more descriptive? Another kind of cleaning occurs in lines 51-59. If you feel like refactoring those lines into their own function too, that'd be welcome, but it's also okay to leave that for another time.
* Please could this function be defined above `FunctionsFramework.http "http_to_bucket" do |request|`, alongside the other helper functions.